### ISSUE TITLE
Unrecognized font family 'serif'

### ISSUE THREAD

**Issue by pxmage** [railsjack/react-native-mathjax-svg#2]
![image](https://user-images.githubusercontent.com/52536798/85011162-bdbfe480-b193-11ea-97a1-1995fa16ba44.png)

I tried to follow the example code
```javascript
 <MathJax
  fontSize={16}
  color={'red'}
  >
  4sen(α)cos^2(α/2)
</MathJax>
```

And then I got this error, any ideas?

**Comment by railsjack** [comment]
Sorry for the delay to response. Let me check right now.

**Comment by Youssouf-Ouedraogo** [comment]
It works for android but the problem persists for iOS.

**Comment by pxmage** [comment]
> It works for android but the problem persists for iOS.

Bro, I have found an alternative `react-native-math-view` while I was waiting for this issue.

Whether you want to use it or not, it's up to you ;)

**Comment by Youssouf-Ouedraogo** [comment]
> > It works for android but the problem persists for iOS.
> 
> Bro, I have found an alternative `react-native-math-view` while I was waiting for this issue.
> 
> Whether you want to use it or not, it's up to you ;)

Thank you very much, bro. I checked it out but my project is an expo project so I am getting an Invariant Violation error. It says that "RNMathView" Was not found in the UIManager.  I wonder if you faced the same problem and how did go about fixing it. 

Again thanks for your help

**Comment by pxmage** [comment]
> > > It works for android but the problem persists for iOS.
> > 
> > 
> > Bro, I have found an alternative `react-native-math-view` while I was waiting for this issue.
> > Whether you want to use it or not, it's up to you ;)
> 
> Thank you very much, bro. I checked it out but my project is an expo project so I am getting an Invariant Violation error. It says that "RNMathView" Was not found in the UIManager. I wonder if you faced the same problem and how did go about fixing it.
> 
> Again thanks for your help

Oh, it requires you to eject, means it only works in bare work flow.

I ejected cuz I need to achieve some IAP/IAB features.
I guess the problem is you are still in the regular expo work flow, because in bare work flow, you need to install some native dependencies.

I would recommend you just eject because it would be way more convenient while you can still enjoy those `expo-xxxx` libraries and embrace the whole react-native ecosystem.

**Comment by railsjack** [comment]
@pxmage Please let me know if it's a font problem or not so that I can add some necessary fonts.

**Comment by Youssouf-Ouedraogo** [comment]
> @pxmage Please let me know if it's a font problem or not so that I can add some necessary fonts.

Yes I am getting the same exact Unrecognized font family error but just for iOS for some reason

**Comment by railsjack** [comment]
Font problem in RN is a general problem which can be fixed by attaching a necessary font.
Serif is a very common fonts for almost platforms and that's why I chose the font of this package.
But, if you don't have the font on your platform, then you can add it. I recommend you to read this article.
https://codewithchris.com/common-mistakes-with-adding-custom-fonts-to-your-ios-app/
If it is not solved then, I can help you in google meeting or something else.

**Comment by Youssouf-Ouedraogo** [comment]
Thank you very much I will take a look at it and see if I can get it to work and keep you posted

**Comment by Youssouf-Ouedraogo** [comment]
> Font problem in RN is a general problem which can be fixed by attaching a necessary font.
> Serif is a very common fonts for almost platforms and that's why I chose the font of this package.
> But, if you don't have the font on your platform, then you can add it. I recommend you to read this article.
> https://codewithchris.com/common-mistakes-with-adding-custom-fonts-to-your-ios-app/
> If it is not solved then, I can help you in google meeting or something else.

I looked into fonts and I realized that by default Android in RN supports the serif font. However, iOS on the other does not. That is probably why I am getting this error. This is the list of font I was looking at [list of font]( https://github.com/react-native-training/react-native-fonts). I looked that [this list](https://fonts.google.com/) to add it as a custom font but I could not find the particular one that your library requires. If you could please help me I would really appreciate it. I do not even mind-changing the font to a default font that works for both as long as I can render the content that is what matters.  Thanks

**Comment by railsjack** [comment]
Ok, let's have a schedule in google meeting. If possible, let me know your gmail, please.

**Comment by Youssouf-Ouedraogo** [comment]
> Ok, let's have a schedule in google meeting. If possible, let me know your gmail, please.

@railsjack Please email me your available time and date at ouedraogoy674@gmail.com and we can schedule a meeting. Thanks

**Comment by Youssouf-Ouedraogo** [comment]
> > > > It works for android but the problem persists for iOS.
> > > 
> > > 
> > > Bro, I have found an alternative `react-native-math-view` while I was waiting for this issue.
> > > Whether you want to use it or not, it's up to you ;)
> > 
> > 
> > Thank you very much, bro. I checked it out but my project is an expo project so I am getting an Invariant Violation error. It says that "RNMathView" Was not found in the UIManager. I wonder if you faced the same problem and how did go about fixing it.
> > Again thanks for your help
> 
> Oh, it requires you to eject, means it only works in bare work flow.
> 
> I ejected cuz I need to achieve some IAP/IAB features.
> I guess the problem is you are still in the regular expo work flow, because in bare work flow, you need to install some native dependencies.
> 
> I would recommend you just eject because it would be way more convenient while you can still enjoy those `expo-xxxx` libraries and embrace the whole react-native ecosystem.

@pxmage Thank you very much for your help. I am trying to keep it as an expo project because with expo it is really easy for me to have "Over the air update" and other than this maths rendering library I am not going to tap into the native features. Again thanks though I really appreciate it.

**Comment by isumesh** [comment]
Hi Guys,
Facing the same issue on iOS. Any luck resolving it ?

**Comment by isumesh** [comment]
Also @railsjack how do you wrap the content ?

**Comment by railsjack** [comment]
This problem came from the fact that it was using Expo.
Expo's main problem is that it throws exception when using custom fonts in react-native-svg which is used by this package.
To avoid this problem, you should not use Expo in future.

**Comment by dadafrank** [comment]
I don't use Expo, but once I type Chinese, I have the same problem

**Comment by railsjack** [comment]
Chinese? In which case do you need to type it?

**Comment by dadafrank** [comment]
like this,   P{拒绝|H0为真} = a
`$$ P\{\mbox{拒绝} \mid H_{0} \mbox{为真}\} = \alpha $$`

**Comment by fanzhiri** [comment]
cause serif is not include by ios at default
so , we need drop the serif.ttf into Xcode , then build . 
it work well at my project , this way can help you too . @pxmage
please check my commit . 
https://github.com/ShaMan123/react-native-math-view/issues/81
